src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/_help.py

```diff
@@ -9,73 +9,76 @@
           type: string
-          short-summary: 'The name of the table to insert the entity into.'
+          short-summary: The name of the table to insert the entity into.
         - name: --entity -e
           type: list
-          short-summary: 'A space-separated list of key=value pairs. Must contain a PartitionKey and a RowKey.'
+          short-summary: A space-separated list of key=value pairs. Must contain a PartitionKey and a RowKey.
+          long-summary: The PartitionKey and RowKey must be unique within the table, and may be up to 64Kb in size. If using an integer value as a key,
+                        convert it to a fixed-width string which can be canonically sorted.
+                        For example, convert the integer value 1 to the string value "0000001" to ensure proper sorting.
         - name: --if-exists
           type: string
-          short-summary: 'Specify what should happen if an entity already exists for the specified PartitionKey and RowKey.'
+          short-summary: Specify what should happen if an entity already exists for the specified PartitionKey and RowKey.
         - name: --timeout
           short-summary: The server timeout, expressed in seconds.
 """
 
 helps['storage blob upload'] = """
     type: command
-    short-summary: Upload a specified file to a storage blob.
-    long-summary: Creates a new blob from a file path, or updates the content of an existing blob, with automatic chunking and progress notifications.
+    short-summary: Upload a file to a storage blob.
+    long-summary: Creates a new blob from a file path, or updates the content of an existing blob with automatic chunking and progress notifications.
     examples:
-        - name: Upload to a blob with all required fields.
+        - name: Upload to a blob.
           text: az storage blob upload -f /path/to/file -c MyContainer -n MyBlob
 """
 
 helps['storage file upload'] = """
     type: command
-    short-summary: Upload a specified file to a file share that uses the standard SMB 3.0 protocol
-    long-summary: Creates or updates an azure file from a source path with automatic chunking and progress notifications.
+    short-summary: Upload a file to a share that uses the SMB 3.0 protocol.
+    long-summary: Creates or updates an Azure file from a source path with automatic chunking and progress notifications.
     examples:
-        - name: Upload to a file share with all required fields.
+        - name: Upload to a local file to a share.
           text: az storage file upload -s MyShare -source /path/to/file
 """
 
 helps['storage blob show'] = """
     type: command
-    short-summary: Returns properties for a named blob in a container in a storage account.
-    long-summary: Blob properties only.  To show contents of a blob, use az storage blob list
+    short-summary: Show properties for a blob in a container.
+    long-summary: To show the contents of a blob, use 'az storage blob list'.
```

Comment:
@troydai is this accurate?  Assuming it is (which I don't think is the case) the command should be quoted with backticks. However, I'm about 98% certain that you can only view the blob contents by downloading it... certainly not by listing it!

Reply:
No. The `storage blob list` shows the blobs in a container, not the content of the blob.